docs/Fields.md

```diff
@@ -25,10 +25,11 @@ export const PostList = (props) => (
 
 All field components accept the following attributes:
 
-* `source`: Property name of your entity to view/edit. This attribute is required.
+* `elStyle`: A style object to customize the look and feel of the field element itself
```

Comment:
I'd prefer is we keep the two styles attributes close to each other, it's easier to understand.